### ISSUE TITLE
issue #69 not fixed

### ISSUE THREAD

**Issue by banana-man95** [wally-rblx/funky-friday-autoplay#70]
it still has the same behavior, i think `PlaybackSpeed` isnt being grabbed properly and `PlaybackSpeed` still stays at 1 regardless of rate because of that (im not sure tho)

**Comment by wally-rblx** [comment]
yeah, still no executors with decent decompilation ability available to use so stuck waiting for now.

**Comment by banana-man95** [comment]
you could try using electron but it's honestly so dogshit i dont think it would help
